packages/rancher-logging/rancher-logging.patch

```diff
@@ -143,4 +143,6 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +
 +global:
 +  systemDefaultRegistry: ""
++
++dockerRoot: "/var/lib/docker/"
```

Comment:
nit: a helper to ensure it ends with a slash incase a user forgets it

Reply:
That is a good idea